test/ansible/watches.yaml

```diff
@@ -3,3 +3,8 @@
   group: test.example.com
   kind: Inventory
   playbook: /opt/ansible/playbooks/inventory.yml
+
+- version: v1alpha1
+  group: test.example.com
+  kind: Collection
+  role: asmacdo.fixture_collection.dummy
```

Comment:
^ nit could not be your name and instead of that be something as `sdkexample`

Reply:
Thats a real collection https://galaxy.ansible.com/asmacdo/fixture_collection

Reply:
Have we not any sdk collection to use instead?
If not, could we add at least a TODO for we create a POC/example of?

Reply:
In the latest push, I've moved the collection into the tree, and the dockerfile now builds and installs.